### ISSUE TITLE
Changing Label does not change label in terms of use page

### ISSUE THREAD

**Issue by vantuyls** [samvera-labs/hyku#1366]
### Descriptive summary

After labels have been added (Dashboard - Settings - Labels), changes to the label are not reflected in the Terms of Use page.

Problem identified on Hyku Demo Site.

### Expected behavior

Change to any label in this interface should cascade out onto all instances of that label elsewhere in the repository.

### Actual behavior

Change to labels does not change text on Terms of Use page.

### Steps to reproduce the behavior

1. set labels as whatever you want. 
2. save labels
3) see terms of use page includes assigned labels
4. change labels
5. see that terms of use page still uses old labels

**Comment by mjgiarlo** [comment]
Yeah, the Terms of Use and Deposit Agreement are now managed as their own ContentBlock-backed Page objects. This may be a UX thing more than a bug. Advice, @hannahfrost @ggeisler?

**Comment by vantuyls** [comment]
right, but setting the labels changes the TOS page to match the labels initially. So one would expect that changing the labels here would also be reflected the way it was the first time around.

**Comment by ggeisler** [comment]
@mjgiarlo I think @vantuyls is correct in saying that what he is reporting is not a UX issue, but a bug.

Changing your labels in Settings > Labels should be reflected in the Terms of Use and Deposit Agreement. 

The connection between them might have been broken when making the Terms of Use and Deposit Agreement their own Page objects, but I don't see a reason why those pages should not continue to get their label definitions from the Settings > Labels entries.

**Comment by mjgiarlo** [comment]
OK.
